### PR TITLE
Add iaas call 2024-02-14

### PR DESCRIPTION
**Pull request by jschoone**

### CODE REVIEW

File: iaas/20240214.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+- see PR [standards/#472](https://github.com/SovereignCloudStack/standards/pull/472)
+    - remove hard-coded least quotas
+    - mention fair-use policy instead
+- Would this work for you? Could this be stabilized soon?
+    - no objections, will proceed to merge this PR, then return to the stabilization proposal
+
+### Future of ceph day-2 tooling (@yeoldegrove)
+
+- 5-10 minutes
+- after the ongoing discussions about weather to choose a the k8s-native approach with rook we did some research and groundwork
```
Comment (gndrmnn) [inline iaas/20240214.md:70]:
```suggestion
- after the ongoing discussions about whether to choose the k8s-native approach with rook we did some research and groundwork
```

File: iaas/20240214.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  - no official solution for a real migration  to rook
```
Comment (gndrmnn) [inline iaas/20240214.md:71]:
```suggestion
  - no official solution for a real migration to rook
```

File: iaas/20240214.md (hunk 1 above)
Comment (fkr) [inline iaas/20240214.md:70]:
```suggestion
- after the ongoing discussions about wether to choose a the k8s-native approach with rook we did some research and groundwork
```

**Comment by fkr** [comment]
thanks @jschoone @fdobrovolny @gndrmnn for your support and facilitation of today’s meeting!

**Review by gndrmnn — APPROVED** [review]

File: iaas/20240214.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+- Please note your handle
+- Please note the amount of time the topic needs in order to be discussed adequately
+- If your topic requires certain people from our community to be around, please make sure they're present for the meeting
+
+### Encryption between user workloads (@fdobrovolny)
+
+- 10 minutes
+- Do we already have experience in doing this?
+- We want to create a plugin into Neutron using eBPF to secure the traffic automatically between virtual machines :bee:
```
[the remark this answers is not in the document]
Comment (jschoone) [reply iaas/20240214.md]: `for` sounds better, thanks :)

**Pull request merged**